src/StatsdClient/IMetricsTimer.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace StatsdClient
+{
+    public interface IMetricsTimer : IDisposable
+    {
+        void AddTag(params string[] additionalTags);
+    }
+}
```

Comment:
Why do we need to add `AddTag` to an interface?

Reply:
Previously the "StartTimer" method returned an IDisposable.  It's now of type IMetricsTimer, with DogStatsdService initialising the concrete MetricsTimer class.  I used an interface just in case someone wanted the ability to mock it.  If the interface isn't necessary I can make a change so that it uses the concrete MetricsTimer everywhere.